packages/EasyCore/src/Doctrine/Type/DateTimeMicrosecondsType.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Doctrine\Type;
+
+use DateTime;
+use DateTimeInterface;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\Type;
+
+final class DateTimeMicrosecondsType extends Type
+{
+    private const TYPE_NAME = 'datetime';
```

Comment:
We can make this const public and reuse in tests for all the `Type::getType('datetime')` and `Type::overrideType('datetime', DateTimeMicrosecondsType::class)` calls. This will make SonarCloud happy about this:
![image](https://user-images.githubusercontent.com/48544017/85710874-623faa80-b6ef-11ea-9cb5-5027131a5c83.png)

Reply:
Done. SonarCloud is happy :)